pkgs/by-name/ti/tideways-cli/package.nix

```diff
@@ -0,0 +1,92 @@
+{
+  stdenvNoCC,
+  lib,
+  fetchurl,
+  curl,
+  common-updater-scripts,
+  writeShellScript,
+  gnugrep,
+  installShellFiles,
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "tideways-cli";
+  version = "1.2.2";
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  src =
+    finalAttrs.passthru.sources.${stdenvNoCC.hostPlatform.system}
+      or (throw "Unsupported platform for tideways-cli: ${stdenvNoCC.hostPlatform.system}");
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/bin
+    cp tideways $out/bin/tideways
+    chmod +x $out/bin/tideways
+
+    installShellCompletion --cmd tideways \
+      --bash <($out/bin/tideways completion bash) \
+      --zsh <($out/bin/tideways completion zsh) \
+      --fish <($out/bin/tideways completion fish)
+
+    runHook postInstall
+  '';
+
+  passthru = {
+    sources = {
```

Comment:
Why is this in `passthru` and not in a `let...in` ?

Reply:
otherwise the `common-updater-scripts` cannot find it.

```
error: attribute 'sources' in selection path 'tideways-cli.sources.x86_64-linux.drvAttrs.outputHash' not found
update-source-version: error: Couldn't evaluate old source hash from 'tideways-cli.sources.x86_64-linux'!
```